### ISSUE TITLE
Support GLES shaders on native / GL Shaders on Android

### ISSUE THREAD

**Issue by madjin** [janusvr/janus#14]
Adding GLES shader support will make chromakey shader for android and native compatible with each other. Here's a test site: https://vesta.janusvr.com/cruzjo/test-chromakey

**Comment by joseph-cruz** [comment]
Partially fixed as of: ed0f30b. GLES shaders will work in native once the Android branch is merged into master. GL shaders in Android is still a TODO.

**Comment by JamesMcCrae** [comment]
Merged android branch.

**Comment by JamesMcCrae** [comment]
Closing this issue.
